app/views/customers/_form.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="container">
+  <%= form_with(model: @customer) do |form| %>
+    <div class="mb-3">
+      <%= form.label :name, class: 'form-label' %>
+      <%= form.text_field :name, class: 'form-control' %>
+    </div>
+
+    <div class="mb-3">
+      <%= form.label :document, class: 'form-label' %>
+      <%= form.text_field :document, class: 'form-control' %>
+    </div>
+
+    <div class="mb-3">
+      <%= form.label :phone, class: 'form-label' %>
+      <%= form.phone_field :phone, class: 'form-control' %>
+    </div>
+
+    <div class="mb-3">
+      <%= form.label :payment_at, class: 'form-label' %>
+      <%= form.date_field :payment_at, class: 'form-control' %>
+    </div>
+
+    <div class="mb-3">
+      <% if @customer&.payment_at %>
```

Comment:
Nao entendi muito bem pq colocou esse if aqui, qual seria a ideia?

Reply:
Foi para que o rails disse que só dava pra renderizar o 'form.date_field ' se houvesse um dado existente, aí eu coloquei esse if para que fosse renderizado mesmo que o valor estivesse nil. Algo assim kkkkk

Reply:
ué, estranho! não era pra precisar disso não, faz mais uns testes ai que acho que vc consegue resolver sem isso

Reply:
aparece esse erro 'undefined method `date' for nil:NilClass
'